controller/node_controller.go

```diff
@@ -357,27 +357,45 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 		}
 		// get disk available size
 		diskInfo, err := nc.getDiskInfoHandler(disk.Path)
+		readyCondition := types.GetDiskConditionFromStatus(diskStatus, types.DiskConditionTypeReady)
 		// if the file system has changed
 		if err != nil {
-			logrus.Errorf("Get disk information on node %v error: %v", node.Name, err)
+			if readyCondition.Status != types.ConditionStatusFalse {
+				readyCondition.LastTransitionTime = util.Now()
+			}
+			readyCondition.Status = types.ConditionStatusFalse
+			readyCondition.Reason = types.DiskConditionReasonDiskIsNotExist
+			readyCondition.Message = fmt.Sprintf("Get disk information on node %v error: %v", node.Name, err)
```

Comment:
`DiskNotExist`.

Or

`DiskDoesNotExist`, which is weirder.

But in fact, the reason should be direct. So `DiskConditionReasonNoDiskInfo`.